aws/lambda/managedeni.py

```diff
@@ -292,6 +292,6 @@ def on_delete(event, context):
                     log.info("Deleting network interface %s" % eni_id)
                 except ClientError as e:
                     reason = "Failed to delete network interface %s" % eni_id
-                    log.exception(reason)
```

Comment:
We still want to log out an exception. `log.exception` will print out the stack trace, which is useful for debugging lambda function.